lib/cucumber/tcl.rb

```diff
@@ -13,9 +13,27 @@ module Cucumber
   module Tcl
 
     def self.install(cucumber_config)
-      create_step_definitions = lambda {
-        StepDefinitions.new(Framework.new(cucumber_config))
-      }
+      # Unless configured off, we should start up a new
+      # framework for each scenarion, which results
+      # in a new TCL interpreter.  This can be used
+      # to check that there is no data leakage between
+      # scenarios when testing poorly understood code
+      if ENV["NEW_INTERPRETER"].nil?
+          new_interpreter = 1
+      else 
+          new_interpreter = ENV['NEW_INTERPRETER'].to_i
+      end
```

Comment:
Otherwise you could negate it, like this:

new_interpreter = (ENV['NEW_INTERPRETER'] != '0')